src/core/annotation.js

```diff
@@ -671,15 +689,7 @@ var TextAnnotation = (function TextAnnotationClosure() {
       this.data.rect[2] = this.data.rect[0] + DEFAULT_ICON_SIZE;
       this.data.name = dict.has('Name') ? dict.get('Name').name : 'Note';
     }
-
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
-    }
-
-    this.data.hasPopup = dict.has('Popup');
-    this.data.title = stringToPDFString(dict.get('T') || '');
-    this.data.contents = stringToPDFString(dict.get('Contents') || '');
+    this._preparePopup(dict);
```

Comment:
Nit: I think that you can now remove the variable and simply use `parameters.dict` here.